src/Components/RequestGeneratorBuilder.js

```diff
@@ -51,7 +51,7 @@ export default class RequestGeneratorBuilder extends preact.Component {
             });
         });
 
-        if (Privacy.isAllowed(PRIVACY_ACTIONS.SAVE_ID_DATA)) this.idData = new IdData();
+        if (Privacy.isAllowed(PRIVACY_ACTIONS.SAVE_ID_DATA)) this.idData = new SavedIdData();
```

Comment:
Ok, so now that I look at the code occurrences of the class again I can see that I missed some things. All variables that contain instances of the class should of course be renamed as well. The confusion is real. So rename this as well please.

Reply:
Just for clarity: Occurences of `id_data` that do not contain an instance of `SavedIdData` should not be renamed, like this one: `this.state.request.id_data`.

Reply:
No worries, i'll look into it.